src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -602,6 +602,20 @@ public function testObjectsSupportDisabledWithExceptions($yaml)
         $this->parser->parse($yaml, Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE);
     }
 
+    public function testCanParseContentWithTrailingSpaces()
+    {
+        $yaml = <<<YAML
+items:  
+  foo: bar
+YAML;
```

Comment:
We should write the fixture this way: `$yaml = "items:  \n  foo: bar";`. Otherwise, someone will save the test file with an editor that strips trailing whitespace which will make the test pretty useless.